src/apps/AppCenter/InstalledApps/AppCardContent.js

```diff
@@ -1,78 +1,124 @@
 import React from 'react'
```

Comment:
Could we consolidate this component with the Card defined in `DiscoverApps`?

I think the main difference that we can control via props are:

- Whether to show the app's link or not
- What Tag to display
- On click behaviour